main.py

```diff
@@ -6,9 +6,26 @@
 import factory
 import loader
 from shape import Rectangle, Circle, Triangle, Parallelogram, Shape
+from classes import ManageTimestamps
+from PIL import Image
+import glob
+import time
+
+print('1')
```

Comment:
I know these were added to debug a CPU 100% issue you were having, don't forget to remove them :)

Reply:
Ok, Will remove them